packages/params/src/constants.ts

```diff
@@ -9,12 +9,10 @@ export const MAX_REQUEST_BLOCKS = 2 ** 10; // 1024
 /** Rationale: https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/p2p-interface.md#why-are-there-attestation_subnet_count-attestation-subnets */
 export const ATTESTATION_SUBNET_COUNT = 64;
 
-// Altair - TODO: Make sure it's not included in the spec
-export const SYNC_COMMITTEE_SUBNET_COUNT = 8;
+// https://github.com/ethereum/eth2.0-specs/blob/v1.1.0-alpha.5/specs/altair/validator.md
+export const SYNC_COMMITTEE_SUBNET_COUNT = 4;
 
 export const MIN_SYNC_COMMITTEE_PARTICIPANTS = 1;
-//~27 hours
-export const LIGHT_CLIENT_UPDATE_TIMEOUT = 2 ** 13;
```

Comment:
The spec removes `MAX_VALID_LIGHT_CLIENT_UPDATES` as well and model `LightClientStore.validUpdates` as a Set instead of a List, which we can't model ssz type accordingly.

So I think just keep it as is?

Reply:
In our lightclient client implementation we don't use neither a set nor a list, just keep the best update overall. So you can delete this constant.

Reply:
it's been used here, don't know how to delete that https://github.com/ChainSafe/lodestar/blob/f6778740075fe2b75edf94d1db0b5691039cb500/packages/types/src/altair/sszTypes.ts#L265

Reply:
You can just give it an arbitrary number since it doesn't matter. You can set it to
```
EPOCHS_PER_SYNC_COMMITTEE_PERIOD * SLOTS_PER_EPOCH
```